### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by debitan**
## This change
- Add in additional steps to make the set up process clearer.

## Possible future enhancement
- Add in how the sandbox works, the need to push changes to master. How to add a tag if your changes are breaking other tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,27 +5,38 @@ Documentation
...
-2. Start sandbox server on port `8001`:
+
+3. Start sandbox server on port `8001`:
 	```bash
 	docker run -it -p 8001:8001 data-hub-sandbox
 	```
 	
 	Stop the server by pressing `CTRL`+`C`
 
+5. You'll also now have to run redis as the sandbox doesn't come with this and will throw an error on the frontend without it. So open a new terminal window and run `docker run -it -p 6379:6379 redis:3.2`
```
Comment (PippoRaimondi) [inline README.md]: I think this information is good but should be added to data-hub-frontend repo, in the functional test section!

Comment (debitan) [reply README.md]: Let's add it to both!

Comment (debitan) [reply README.md]: Reason being that you need this step to actually get the sandbox up and running and this is what this readme is supposed to help you do.

Comment (debitan) [reply README.md]: Hi @PippoRaimondi. Is it ok to keep this part in given the reasoning above?

**Pull request merged**